compiler/src/Language/Bond/Codegen/Java/Class_java.hs

```diff
@@ -40,9 +40,11 @@ package #{javaPackage};
 #{Java.generatedClassAnnotations}
 public class #{declName}#{params}#{maybe interface baseClass structBase} {
 
+    public static final com.microsoft.bond.StructBondType<#{declName}> BOND_TYPE = null;
```

Comment:
Yes, it needs to be public for static access.

---
In reply to: [124081480](https://github.com/Microsoft/bond/pull/502#discussion_r124081480) [](ancestors = 124081480)